lib/screens/screens_by_alert/self_refresh_runner.ex

```diff
@@ -0,0 +1,102 @@
+defmodule Screens.ScreensByAlert.SelfRefreshRunner do
+  @moduledoc """
+  A "job-runner" GenServer that routinely checks for out-of-date
+  screen data, and simulates data requests for any that it finds.
+  """
+
+  alias Screens.ScreensByAlert
+
+  # (Not a real module--just a name assigned to the Task.Supervisor process that supervises each simulated data request run)
+  alias Screens.ScreensByAlert.SelfRefreshRunner.TaskSupervisor
+
+  require Logger
+  use GenServer
+
+  @type state :: %__MODULE__{}
+
+  # The server is not stateful.
+  defstruct []
```

Comment:
I think I was following the informal best practice of having GenServers always use a struct as state, but yeah, the state could just be `nil`. I guess the idea was that if we do need to start keeping track of some state values in the future, it would be easier to just add some new keys to the existing struct.

I'll change the state to `nil` for now.